packages/ord-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -70,50 +70,53 @@ export function SelectWalletModal({
     [disconnectWallet],
   );
 
-  const onConnectUnisatWallet = async (readOnly?: boolean) => {
-    const listener = () => {
-      onConnectUnisatWallet();
-    };
-    try {
-      window.unisat.removeListener("accountsChanged", listener);
-    } catch (_) {
-      // This will fail on first run, handle it silently
-    }
```

Comment:
not necessary to have this anymore because we check whether the extension is ready before reaching here. Also, adding/removing listeners are now only within the effect